src/server.ts

```diff
@@ -38,6 +40,23 @@ class Server {
       this.httpsApp = express();
     }
     this.app = Config.httpsEnabled ? this.httpsApp : this.httpApp;
+    if (Config.pingEnabled) {
+        this.app.use(health.ping('/' + Config.pingEndpoint));
+    }
+    if (Config.logRequests) {
```

Comment:
I would opt to enable the "daily" property in order to get a logfile per day.

Reply:
Totally! As you can see from the commit comment, that's what I intended. I guess somewhere things got fuzzy, and it got left out.